Zotero/Scenes/Detail/PDF/Views/PDFDocumentViewController.swift

```diff
@@ -216,7 +216,7 @@ final class PDFDocumentViewController: UIViewController {
     }
 
     private func update(state: PDFReaderState, pdfController: PDFViewController) {
-        if state.changes.contains(.interfaceStyle) {
+        if state.changes.contains(.appearanceMode) {
```

Comment:
Previous check should remain, otherwise while on Automatic it misses system change in appearance mode.
```suggestion
        if state.changes.contains(.interfaceStyle) || state.changes.contains(.appearanceMode) {
```